react-gbajs/src/emulator/util.js

```diff
@@ -244,3 +244,33 @@ Serializer.pointer.prototype.readString = function(view) {
 	}
 	return bytes.join('');
 };
+
+window.isPrimitive = function isPrimitive(val) {
```

Comment:
slapped several things onto window due to not fully getting how react-gbajs works. Just wanted to get things working quickly.